### ISSUE TITLE
Formatting issues

### ISSUE THREAD

**Issue by macbaszii** [nicklockwood/SwiftFormat#15]
```swift
init?<T: UIViewController where T: protocol<WZStatusCallback, WZAudioSink, WZVideoSink>>(withViewController viewController: T) {
    // Initialization Code
}
```

In this case (Failable Initialization), SwiftFormat will make
1. space around `?` and you will get compile error. (strict by syntax)
2. space after < and before > in my generic T type. I think SwiftFormat see that symbol as greater than and lower than mark. So, I think it should be ignored in this case.

**Comment by nicklockwood** [comment]
Thanks, I'll get those fixed.

**Comment by nicklockwood** [comment]
Fixed in 0.7.1
